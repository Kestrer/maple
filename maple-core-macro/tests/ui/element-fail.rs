use maple_core::prelude::*;

fn compile_fail() {
    template! { p.my-class#id };

    template! { button(disabled) };
    template! { button(on:click) };

    template! { button(a.b.c="123") };
}

fn main() {}
