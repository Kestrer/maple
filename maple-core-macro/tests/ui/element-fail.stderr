error: unexpected token
 --> $DIR/element-fail.rs:4:18
  |
4 |     template! { p.my-class#id };
  |                  ^

error: expected an assignment expression
 --> $DIR/element-fail.rs:6:24
  |
6 |     template! { button(disabled) };
  |                        ^^^^^^^^

error: expected an assignment expression
 --> $DIR/element-fail.rs:7:24
  |
7 |     template! { button(on:click) };
  |                        ^^^^^^^^

error: unexpected token
 --> $DIR/element-fail.rs:9:24
  |
9 |     template! { button(a.b.c="123") };
  |                        ^^^^^
